product_docs/docs/pem/9/reports.mdx

```diff
@@ -22,9 +23,23 @@ Only superusers or the users with the pem_admin role can download the System Con
 
 Information in these reports shows the latest probe run time.
 
-## System Configuration Report
+## Alert History report
 
-The System Configuration Report provides detailed information about the PEM Agents group, PEM Server Directory group, and custom groups listed under the browser tree. These groups can contain Postgres Enterprise Manager, PEM agent, and database servers. You can download this report in HTML and JSON formats.
+The Alert History report provides detailed information about the alerts history at the agent or server level in JSON format.
+
+Select **Management > Reports > Alert History Report**. A dialog box opens:
```

Comment:
I don't think you need this image. Instead I would take out "A dialog box opens" and, before the list of options, add:

From the dialog box, select your options: